functions_test.go

```diff
@@ -33,3 +33,16 @@ func ExampleMM() {
 	_ = z
 	// Output:
 }
+
+func ExampleRelu() {
```

Comment:
This should be `TestRelu(t *testing.T)` instead of `ExampleRelu()` because we need to check the returned values, right?